modules/elastodyn/src/ElastoDyn.f90

```diff
@@ -6591,6 +6396,61 @@ SUBROUTINE Teeter( t, p, TeetDef, TeetRate, TeetMom )
    RETURN
 END SUBROUTINE Teeter
 !----------------------------------------------------------------------------------------------------------------------------------
+!> This routine computes the Yaw Friction Torque due to yaw rate and acceleration.
+SUBROUTINE YawFriction( t, p, Fz, Mzz, Omg, OmgDot, YawFriMf )
+!..................................................................................................................................
+
+      ! Passed Variables:
+   REAL(DbKi), INTENT(IN)             :: t                                       !< simulation time
+   TYPE(ED_ParameterType), INTENT(IN) :: p                                       !< parameters from the structural dynamics module
+   REAL(R8Ki), INTENT(IN )            :: Fz, Mzz                                 !< The teeter deflection, x%QT(DOF_Teet).
+   REAL(R8Ki), INTENT(IN )            :: Omg                                !< The yaw rate (rotational speed), x%QDT(DOF_Yaw).
+   REAL(R8Ki), INTENT(IN )            :: OmgDot                             !< The yaw acceleration (derivative of rotational speed), x%QD2T(DOF_Yaw).
+   
+   REAL(ReKi), INTENT(OUT)            :: YawFriMf                                 !< The total friction torque (Coulomb + viscous).
+
+      ! Local variables:
+   REAL(ReKi)                         :: temp                                   ! It takes teh value of Fz or -1.
+
+   
+   SELECT CASE ( p%YawFrctMod  ) ! Which friction model are we using? 0=None, 1=does not use Fz, 2=does use Fz
```

Comment:
Compare with documentation and descriptions in the input files on `YawFrctMod`.

Reply:
Compared and checked